### ISSUE TITLE
Regression with 4.8.4 on Node 18.3

### ISSUE THREAD

**Issue by dilyanpalauzov** [telegraf/telegraf#1636]
Upgrading to 4.8.4 and runnig my code, I get
```
node:internal/errors:466
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in /git/zzz/node_modules/telegraf/package.json imported from /git/zzz/src/telegram.js
    at new NodeError (node:internal/errors:377:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:440:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:664:7)
    at packageResolve (node:internal/modules/esm/resolve:893:14)
    at moduleResolve (node:internal/modules/esm/resolve:959:20)
    at defaultResolve (node:internal/modules/esm/resolve:1174:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:605:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:318:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}

Node.js v18.3.0
```
Downgrading to 4.8.3 fixes the problem.

**Comment by MKRhere** [comment]
Hi, thanks for reporting. Could you verify for me that this works with `telegraf@4.8.5-canary.1`? If it works, I'll push update 4.8.5 to stable.

**Comment by dilyanpalauzov** [comment]
`telegraf@4.8.5-canary.1` does work.

**Comment by MKRhere** [comment]
Fixed in 87cb1b9ec68774e5f9abafe355bc3513a54c847f, sailing off to npm telegraf@4.8.5 🎉
